### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by ulisesedu7**
In this project I've created a concert page based on the design of the capstone project no. 1. 
- I used HTML, CSS & Javascript to built it. 
- Along the way I worked with Git and Github and several resources among the web to make everything accordingly.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+            <li><a class="mobile-menu-a main-logo" href="index.html"><img src="images/tchaikovski-img.jpg"  alt="An image of Tchaikovski"></a></li>
+            <li><a class="mobile-menu-a" href="about-page.html" id="about-link-m">About</a></li>
+            <li><a class="mobile-menu-a" href="#program-section">Program</a></li>
+            <li><a class="mobile-menu-a" href="#repertoire">Repertoire</a></li>
+            <li><a class="mobile-menu-a" href="#sponsors">Sponsors</a></li>
+            <li><a class="mobile-menu-a" href="#">News</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
```
Comment (aimemalaika) [inline index.html:62]: - [x] Kindly add the dark top bar containing social media links, logout, and language links aligned to the right on top of the menus
Screen Shot 2022-05-20 at 07 49 56

Your design
Screen Shot 2022-05-20 at 07 50 18

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @ulisesedu7, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi @ulisesedu7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it ✔ 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:62]: Addressed ✔

**Pull request merged**